packages/flx/pubspec.yaml

```diff
@@ -6,7 +6,7 @@ homepage: https://github.com/flutter/flutter/tree/master/packages/flx
 dependencies:
   bignum: ^0.0.7
   asn1lib: ^0.4.1
-  cipher: ^0.7.1
+  pointycastle: ^0.10.0-dev
```

Comment:
Yep! You're right. I switched to a newer version of bignum, and it works now. Thanks.